scripts/issue/1489/assoc_and_treasury_council_funding.py

```diff
@@ -0,0 +1,105 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+from brownie import accounts
+
+# flag
+PROD = False
+COEF = 0.99
+DEADLINE = 60 * 60 * 4
+
+# funding figures (treasury)
+TREASURY_COUNCIL_YEARLY_FUNDING = 300_000e6
```

Comment:
this assume `USDC` precision of 6?

Reply:
It does, of course it tends to be better to fetch decimals from the source but in this case I think it is safe to hardcode.